app.js

```diff
@@ -25,13 +27,27 @@ function renderPageContent(codePanel$, specsPanel$) {
 }
 
 function app(sources) {
+    sources.exercise = sources.HTTP
```

Comment:
If you check Network tab on chrome, you can see that this approach issues two http requests on the front-end side. We can address keeping context in separate PR, but let's limit it to only one request for now.